testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/ConfigurationTools.cs

```diff
@@ -0,0 +1,92 @@
+using UnityEngine;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text;
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class ConfigurationTools
+    {
+        public static JobQueueItemArray GetRemoteConfig()
+        {
+            var theList = new JobQueueItemArray();
+            try
+            {
+                using var client = new HttpClient();
+                var cancelAfterDelay = new CancellationTokenSource(TimeSpan.FromSeconds(10));
+                var responseTask = client.GetAsync("https://multiprocess-log-event-manager.cds.internal.unity3d.com/api/JobWithFile",
+                    HttpCompletionOption.ResponseHeadersRead, cancelAfterDelay.Token);
+                responseTask.Wait();
+                var response = responseTask.Result;
+                var contentTask = response.Content.ReadAsStringAsync();
+                contentTask.Wait();
+                MultiprocessLogger.Log($"remoteConfig content is {contentTask.Result}");
+                JsonUtility.FromJsonOverwrite(contentTask.Result, theList);
+                MultiprocessLogger.Log($"remoteConfig content is {theList.JobQueueItems.Count}");
+
+            }
+            catch (Exception e)
+            {
+                MultiprocessLogger.Log($"GetRemoteConfig - Exception {e.Message}");
+            }
+            finally
+            {
+                MultiprocessLogger.Log($"GetRemoteConfig - finally block");
+            }
```

Comment:
is this needed?

Reply:
Mostly for trace purposes, there's been a few cases where the test hangs and I've been trying to leave enough of a trace to find it. Once I do, this won't need to be here any more.

Reply:
I took it out.